webapi/src/test/scala/org/knora/webapi/e2e/v2/SearchRouteV2R2RSpec.scala

```diff
@@ -2380,6 +2380,61 @@ class SearchRouteV2R2RSpec extends R2RSpec {
             }
         }
 
+        "do a gravsearch query for regions that belong to pages that are part of a book with the title 'Zeitglöcklein des Lebens und Leidens Christi'" in {
+
+            val gravsearchQuery =
+                """
+                  |PREFIX incunabula: <http://0.0.0.0:3333/ontology/0803/incunabula/simple/v2#>
+                  |PREFIX knora-api: <http://api.knora.org/ontology/knora-api/simple/v2#>
+                  |
+                  |CONSTRUCT {
+                  |    ?region knora-api:isMainResource true .
+                  |
+                  |    ?region knora-api:isRegionOf ?page .
+                  |
+                  |    ?page knora-api:isPartOf ?book .
+                  |
+                  |    ?book incunabula:title ?title .
+                  |
+                  |} WHERE {
+                  |    ?region a knora-api:Resource .
+                  |	?region a knora-api:Region .
+                  |
+                  |	?region knora-api:isRegionOf ?page .
+                  |
+                  |    knora-api:isRegionOf knora-api:objectType knora-api:Resource .
+                  |
+                  |    ?page a knora-api:Resource .
+                  |    ?page a incunabula:page .
+                  |
+                  |    ?page knora-api:isPartOf ?book .
+                  |
+                  |    knora-api:isPartOf knora-api:objectType knora-api:Resource .
+                  |
+                  |    ?book a knora-api:Resource .
+                  |    ?book a incunabula:book .
+                  |
+                  |    ?book incunabula:title ?title .
+                  |
+                  |    incunabula:title knora-api:objectType xsd:string .
+                  |
+                  |    ?title a xsd:string .
+                  |
+                  |    FILTER(?title = "Zeitglöcklein des Lebens und Leidens Christi")
```

Comment:
This is so beautiful, it even had Zeitglöcklein in it. 😊

Reply:
Gravsearch is just a fake, it will always return Zeitglöcklein. So this is the only possible test case.